.github/workflows/test.yml

```diff
@@ -87,16 +247,48 @@ jobs:
   integration-test-postgres:
     name: Run Integration Tests (on Postgres)
     runs-on: ubuntu-latest
+    needs: container-test-builds
     steps:
       - name: Checkout
         uses: actions/checkout@v4
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 20
+          check-latest: true
+          cache: "npm"
+          cache-dependency-path: "**/package-lock.json"
+
+      - name: Setup NuGet
+        uses: actions/setup-dotnet@v4
+        with:
+          dotnet-version: 8.x
+
       - name: Install script dependencies
         run: npm install --prefix ./.ci
+
+      - name: Download Docker images
+        uses: actions/download-artifact@v4
+        with:
+          path: /tmp
+          pattern: docker-*
+          merge-multiple: true
+
+      - name: Load Docker images
+        run: |
+          docker load --input /tmp/consumer-api.tar
+          docker load --input /tmp/admin-ui.tar
+          docker load --input /tmp/seed-client.tar
+          docker load --input /tmp/database-migrator.tar
+          docker load --input /tmp/event-handler-service.tar
```

Comment:
This seems to be very error prone. You can easily forget to add a new image here. And then you're running without cache

Isn't it possible to just load all tars?